.gitignore

```diff
@@ -66,6 +66,9 @@ node_modules/
 codekit-config.json
 .pycharm_helpers/
 
+.project
+.pydevproject
```

Comment:
That's just what I thought when I joined the team, too.  Ask Carson whether we have a preference as a team.